portal-web/test/functional/com/liferay/portalweb/tests/poshi_tests/poshi_test01.testcase

```diff
@@ -0,0 +1,186 @@
+        var fieldTextField = FormFields.getFieldName();
+
+        Form.editFieldLabelMetal(fieldValue = "Full Name");
+
+        Form.closeSidebar();
+
+        Form.gotoAddField(
+			fieldPositionNumber = 1,
+			fieldType = "Numeric");
+
+        var fieldNumericField = FormFields.getFieldName();
+
+        Form.editFieldLabelMetal(fieldValue = "Age");
+
+        Form.closeSidebar();
+
+        Form.addTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+        FormViewBuilder.editFieldLabel(
+			fieldLabel = "Full Name",
+			label = "Nome Completo");
+
+        Form.closeSidebar();
+
+        FormViewBuilder.editFieldLabel(
+			fieldLabel = "Age",
+			label = "Idade");
+
+        Form.closeSidebar();
+
+        Form.save();
+
+        Form.publishForm();
+
+        FormsAdminNavigator.gotoPublishedForm();
+
+        Pause(locator1 = 3000);
+
+        FormViewBuilder.viewTranslationLanguage(symbolName = "en-US");
+
+        FormViewBuilder.validateFieldLabel(fieldLabel = "Full Name");
+
+        FormViewBuilder.validateFieldLabel(fieldLabel = "Age");
+
+       FormFields.viewTextField(
+			fieldLabel = "Full Name",
+			fieldName = "Full Name");
+
+		FormFields.editText(
+			fieldName = "Full Name",
+			fieldValue = "Abhner Ramos");
+
+         FormFields.viewTextField(
+			fieldLabel = "Age",
+			fieldName = "Age");
+
+		FormFields.editText(
+			fieldName = "Age",
+			fieldValue = 18);
+
+        FormPortlet.submitSuccessfully();
+
+        Form.submitAgain();
+
+        Pause(locator1 = 2000);
+         
+        FormViewBuilder.viewTranslationLanguage(symbolName = "en-US");
+
+        FormViewBuilder.validateFieldLabel(fieldLabel = "Full Name");
+
+        FormViewBuilder.validateFieldLabel(fieldLabel = "Age");
+
+        FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "português-Brasil");
+
+        FormViewBuilder.validateFieldLabel(fieldLabel = "Nome Completo");
+
+        FormViewBuilder.validateFieldLabel(fieldLabel = "Idade");
+
+         FormFields.viewTextField(
+			fieldLabel = "Nome Completo",
+			fieldName = "Nome Completo");
+
+		FormFields.editText(
+			fieldName = "Nome Completo",
+			fieldValue = "Jerson Oliveira");
+
+         FormFields.viewTextField(
+			fieldLabel = "Idade",
+			fieldName = "Idade");
+
+		FormFields.editText(
+			fieldName = "Idade",
+			fieldValue = 950);
+        
+        FormPortlet.submitSuccessfully();
+
+        Navigator.openURL();
+
+        ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+        FormsAdminNavigator.gotoViewEntries();
+
+        LexiconEntry.viewEntryName(rowEntry = "Abhner Ramos");
+
+        LexiconEntry.viewEntryContent(
+			rowAny = "18",
+			rowEntry = "Abhner Ramos");
+
+        LexiconEntry.viewEntryName(rowEntry = "Jerson Oliveira");
+
+        LexiconEntry.viewEntryContent(
+			rowAny = "950",
+			rowEntry = "Jerson Oliveira");
+
+    }
+}
+
+}
+
```

Comment:
Please send me a new pr without this testcase file and without the path file just above. For that, you must create a branch from master.

Reply:
This branch that I sent the tests is not the master, it's a test branch, but I had already added these other tests before, but I'll solve it.